components/partials/OverviewInfo.tsx

```diff
@@ -11,17 +27,20 @@ export const OverviewInfo = (chapter: Chapter) => {
       <h2 className="text-left text-3xl text-white md:text-5xl">
         {chapter.title}
       </h2>
-      <TabGroup
-        chapter={chapter}
-        items={[
-          {
-            text: 'Info',
-          },
-          {
-            text: 'Challenges',
-          },
-        ]}
+      <Tabs
+        items={tabData}
+        activeId={activeTab}
+        onChange={setActiveTab}
+        classes="mt-8"
       />
+      <div className="flex grow">
+        {activeTab == 'info' &&
+          <Overview chapter={chapter}></Overview>
+        }
+        {activeTab == 'challenges' && (
+          <ChallengeList challenges={chapter.lessons} path={chapter.slug} ></ChallengeList>
```

Comment:
I know we've gone over this... a lesson can be a challenge?

Overall, I'm a big fan of the naming changes you've introduced here. Really like `ChallengeList` and `ChallengeItem`. they are very explicit.

Reply:
I'll merge this one in. Feel free to create another PR for your proposed name change, it might not affect just this component but also other files (for example, pages can have a contentType of `challenge`). I am still confused about what would be a lesson and what would be a challenge (did we write that down somewhere and I just didn't read thoroughly).

Reply:
All good, my comment was more of a question. Trying to get clarification to make sure I understand things correctly :)